tests/test_dynamical_decoupling.py

```diff
@@ -575,6 +575,42 @@ def test_conversion_of_pulses_with_arbitrary_detuning_rotations():
     assert _all_greater_or_close(driven_control.duration, minimum_segment_duration)
 
 
+def test_conversion_of_tightly_packed_sequence():
+    """
+    Tests if the method to convert a DDS to driven controls handles properly
+    a sequence tightly packed with pulses, where there is no time for a gap between
+    the pi/2-pulses and the adjacent pi-pulses.
+    """
+    # create a sequence containing 30 pi-pulses and 2 pi/2-pulses at the extremities
```

Comment:
Similar to the comment on the last PR, but I think we'd get the same amount of coverage with a sequence of maybe 4 offsets (pi/2, pi, pi, pi/2). In the context of the bug Luigi hit it's nice to verify that situation exactly, but for the test that we actually merge I'd suggest favouring simplicity.